incompatibility with extension quarkus-smallrye-reactive-messaging-rabbitmq

Hi,
I added web bundler to my existing project and got errors during quarkusDev as long as 

`    
implementation 'io.quarkus:quarkus-smallrye-reactive-messaging-rabbitmq'
`
was on my classpath.

I created a new small app with codestart with minimal dependencies and could reproduce this in any case.

### Environment: 

- quarkus 3.11.0 
- java 17
- gradle  8.6 or 8.7 (groovy)
- web-bunder 1.15.0, 1.15.1 and 1.15.2
### Stacktrace:

`Caused by: java.lang.ExceptionInInitializerError
	at io.quarkiverse.web.bundler.deployment.DevModeBundlingProcessor.watch(DevModeBundlingProcessor.java:108)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:849)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
	at java.base/java.lang.Thread.run(Thread.java:840)
	at org.jboss.threads.JBossThread.run(JBossThread.java:483)
Caused by: java.lang.NullPointerException: Make sure the version.properties contains 'esbuild.version'.
	at java.base/java.util.Objects.requireNonNull(Objects.java:235)
	at io.mvnpm.esbuild.Bundler.resolveEmbeddedVersion(Bundler.java:34)
	at io.mvnpm.esbuild.Bundler.<clinit>(Bundler.java:21)
	... 13 more
`

--- a/pom.xml
+++ b/pom.xml
@@ -30,7 +30,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <quarkus.version>3.9.0</quarkus.version>
-    <esbuild-java.version>1.4.2</esbuild-java.version>
+    <esbuild-java.version>1.4.3</esbuild-java.version>
     <scrimage-core.version>4.1.1</scrimage-core.version>
     <yuicompressor.version>2.4.8</yuicompressor.version>
   </properties>

